stdlib/oauth2/src/main/ballerina/src/oauth2/outbound_oauth2_provider.bal

```diff
@@ -597,12 +605,12 @@ function prepareRequest(RequestConfig config) returns http:Request|Error {
     return req;
 }
 
-# Extract the access token from the JSON payload of a given HTTP response and update the token cache.
+# Extracts the access token from the JSON payload of a given HTTP response and update the token cache.
 #
 # + response - HTTP response object
 # + oauth2CacheEntry - OAuth2 cache entry
 # + clockSkewInSeconds - Clock skew in seconds
-# + return - Extracted access token or `Error` if an error occurred during the HTTP client invocation
+# + return - Extracted access token or `oauth2:Error` if an error occurred during the HTTP client invocation
```

Comment:
```suggestion
# + return - Extracted access token or `oauth2:Error` if an error occurred during the HTTP client invocation
```
Please refer the template for the correct format.

Reply:
```suggestion
# + return - Extracted access token or else `oauth2:Error` if an error occurred while HTTP client invocation
```